lib/src/int64.dart

```diff
@@ -668,6 +668,14 @@ class Int64 implements IntX {
     return hexStr;
   }
 
+  /**
+   * Returns the digits of [this] when interpreted as an unsigned 64-bit value.
+   */
+  String toStringUnsigned([int radix]) {
```

Comment:
Is there *any* chance we'll want to add other params to this later? Should this be positional?

Reply:
I removed the parameter and added toRadixStringUnsigned() to be consistent with the existing methods.